src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -45,7 +49,7 @@ export class TerminalInstance {
 		this.xterm.on('data', (data) => {
 			this.terminalProcess.process.send({
 				event: 'input',
-				data: data
+				data: data && data.replace(TerminalInstance.eolRegex, os.EOL)
```

Comment:
This should be split out above for clarify, it looks like it's passing in a boolean right now.